src/game/server/instagib/round_stats.cpp

```diff
@@ -123,21 +123,70 @@ void IGameController::GetRoundEndStatsStrPsv(char *pBuf, size_t Size)
 	int ScoreLimit = m_GameInfo.m_ScoreLimit;
 	int TimeLimit = m_GameInfo.m_TimeLimit;
 
+	std::optional<std::string> RedClan = std::nullopt;
```

Comment:
I personally don't like `std::` that much. And the code base mostly uses C strings. I see no advantage of the std::string here it just makes the code more complicated because it has to be casted from and to C string.

I am also pretty sure its also slower performance wise than just to use a C string.

The code is not too performant relevant so its fine. But next time no std::string please :p unless it has an actual advantage over the C string.

Reply:
Well, I find no actual reason for std::string here too, but std::optional does not work with c-arrays, and if I didn't use std::optional, I had to make 2 more bool variables. That probably would have greater performance and memory usage, but this looks kinda cool.

Reply:
Could he just `const char *pClan` no need for optional you can just make it a null pointer.

Reply:
Mhm.... true.... looks like I'm dum 😳.